data/korath/korath havens.txt

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2020 by Saugia
```

Comment:
@Saugia been time travelling?

Reply:
That sounds like something a time traveler would say.

Also, could put Terin's name in this file too since they helped me with a lot of it.